src/app/models/recentActivity.ts

```diff
@@ -0,0 +1,25 @@
+export class RecentActivity {
```

Comment:
Thanks for adding in this model for RecentActivity! Maybe we can get more use out of it in the frontend going forward